.github/workflows/aarch64.yml

```diff
@@ -25,7 +25,13 @@ jobs:
         install: |
           export CARGO_TERM_COLOR=always
           export CARGO_REGISTRIES_CRATES_IO_PROTOCOL=sparse
-          apt-get update && apt-get install -y gdb pip curl python3.10-dev clang llvm build-essential
+          apt-get update && apt-get install -y gdb pip curl wget python3.10-dev clang llvm build-essential
+          wget https://download.visualstudio.microsoft.com/download/pr/092bec24-9cad-421d-9b43-458b3a7549aa/84280dbd1eef750f9ed1625339235c22/dotnet-sdk-8.0.101-linux-arm64.tar.gz
+          mkdir /dotnet && tar zxf dotnet-sdk-8.0.101-linux-arm64.tar.gz -C /dotnet && rm -rf dotnet-sdk-8.0.101-linux-arm64.tar.gz
```

Comment:
Do we need to install dotnet on aarch64? Are there any tests that use dotnet on arm?

Reply:
You can try to run them, but if they take a long time, it will be better to disable them.

Reply:
i decided not to run these tests on arm (there are no specific cases that must me checked on arm)